datafusion/proto/src/common.rs

```diff
@@ -17,26 +17,17 @@
 
 use datafusion_common::{DataFusionError, Result};
 
-pub fn csv_delimiter_to_string(b: u8) -> Result<String> {
-    let b = &[b];
-    let b = std::str::from_utf8(b)
-        .map_err(|_| DataFusionError::Internal("Invalid CSV delimiter".to_owned()))?;
-    Ok(b.to_owned())
-}
-
-pub fn str_to_byte(s: &String) -> Result<u8> {
+pub fn str_to_byte(s: &String, flag: &str) -> Result<u8> {
```

Comment:
I know this PR didn't introduce this change, but `str_to_bytes` doesn't need to be `pub`:

```suggestion
fn str_to_byte(s: &String, flag: &str) -> Result<u8> {
```

Reply:
it might also be clearer if the name of `flag` was something like `description`